platform_versioned_docs/version-23.4.0/launch/launchpad.mdx

```diff
@@ -5,95 +5,96 @@ date: "21 Apr 2023"
 tags: [launchpad]
 ---
 
-Seqera Platform Launch makes it easy for users without Nextflow expertise to configure pipeline parameters and launch workflows. View, configure, and launch pipelines on the **Launch Pipeline** page, accessed from your workspace **Launchpad**.
+View, configure, and launch pipelines from your workspace **Launchpad**.
 
 ## Launchpad
 
-The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** drop-down to sort pipelines, either by name or most-recently updated.
+The **Launchpad** enables workspace users to launch pre-configured pipelines, add new pipelines, or perform a quick launch of unsaved pipelines. Use the **Sort by:** dropdown to sort pipelines, either by name or most-recently updated.
 
 :::note
 A pipeline is a repository containing a Nextflow workflow, a compute environment, and pipeline parameters.
 :::
 
-The list layout is the default **Launchpad** view. Use the toggle to the right of the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
+The list layout is the default **Launchpad** view. Use the toggle next to the **Search** field to switch between the list and tile views. Both views display the compute environment of each pipeline for easy reference.
 
 ## Launch form
 
-The **Launch form** is used to launch pipelines and to add pipelines to the Launchpad. Select **Launch** next to a saved pipeline in the Launchpad list, or select **launch a run without configuration** below the Launchpad heading to perform a quick launch of an unsaved pipeline.
+The launch form is used to launch pipelines and to add pipelines to the **Launchpad**. Select **Launch** next to a saved pipeline in the list, or select **launch a run without configuration** to perform a quick launch of an unsaved pipeline.
 
-**Launch a saved pipeline**
+The launch form consists of ([General config](#general-config), [Run parameters](#run-parameters), and [Advanced options](#advanced-options)) sections to specify your run parameters before execution, and an execution summary. Use section headings or select the **Previous** and **Next** buttons at the bottom of the page to navigate between sections. 
 
-Select **Launch** next to the pipeline of your choice to open the pipeline launch form.
+For saved pipelines, **General config** and **Run parameters** fields are prefilled and can be edited before launch. 
 
-Seqera uses a `nextflow_schema.json` file in the root of the pipeline repository to dynamically create a form with the necessary pipeline parameters.
+### General config 
 
-All pipelines contain at least these parameters:
+- **Pipeline to launch**: A Git repository name or URL. For saved pipelines, this is prefilled and cannot be edited. Private repositories require [access credentials](../credentials/overview.mdx).
```

Comment:
I always wonder whether it is Git or git. Based on this it is Git: https://github.com/swcarpentry/git-novice/issues/505